docs/openj9_support.md

```diff
@@ -85,9 +81,9 @@ OpenJDK 8 binaries are expected to function on the minimum operating system leve
 - Not all of these distributions are tested, but the following distributions are expected to function without problems:
 
     - Linux distributions that have a minimum glibc version 2.12 (x) or 2.17 (others)
-    - RHEL 9.0 and Ubuntu 22.04 distributions on AArch64 architecture
+    - RHEL 9.2 and Ubuntu 22.04 distributions on AArch64 architecture
 
-- Support for OpenJ9 on CentOS 6.10 might be available even though CentOS 6 is no longer supported by the CentOS project. However, this support is deprecated and will be removed in a future release. With the removal of support, the minimum glibc version (x) will be increased from 2.12 to 2.17.
+- Support for OpenJ9 on CentOS 6.10 is removed from 0.46.0 release onwards. With the removal of support, the minimum glibc version (x) is increased from 2.12 to 2.17.
```

Comment:
The minimum glibc level doesn't change at this time even though we are removing support for CentOS 6. We can state the minimum glibc level may change at a future time.

Reply:
Should we specifically say that "...the minimum glibc version (x) might be increased from 2.12 to 2.17 in future releases" or should we keep it generic "...With the removal of support, the minimum glibc version (x) might change in future releases."?

Thanks!

Reply:
Sure, keep it generic so we have more flexibility.